### ISSUE TITLE
Some macOS plist files not showing correctly

### ISSUE THREAD

**Issue by Aerozolic** [danielbayley/atom-plist-converter#3]
Initial issue over Atom: https://github.com/atom/atom/issues/15579

At first you could test out the following file on your own mac. Let me know if it works fine for you.
/Library/Preferences/com.apple.windowserver.plist

**Comment by Metal-Snake** [comment]
I just fixed this in my fork. Needs a bit more testing though.

If you want to try it, you can find it here: https://github.com/Diskutant/atom-plist-converter

**Comment by Aerozolic** [comment]
@Diskutant Few of the plists that weren't showing correctly before are now working.

**Comment by Metal-Snake** [comment]
So there are still many plists that don't show correctly? Do you have an example file?

**Comment by Aerozolic** [comment]
@Diskutant I mean the few plist files that I previously found that didn't work now work. So I guess it might be fixed.

**Comment by danielbayley** [comment]
@Diskutant I took a look at your code, thanks for trying to fix this.

I tested a bunch of files with that approach, including the one you mentioned @Aerozolic. But basically doing this asynchronously seems to be problematic in general.

I have rewritten it locally, making use of [_simple-plist_](https://github.com/wollardj/node-simple-plist), and have it working much better now. I'll try with `async`/`await` also.
